usr/local/sbin/lncm-configinvoicer.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+'''
+Invoicer configuration Script
+
+Copyright © 2018-2019 LNCM Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+How to use
+
+Dependencies
+sudo pip3 install toml
+
+Executing
+./lncm-configinvoicer.py <rpcusername> <rpcpassword>
+
+Todo
+Make it write anything to the config file. For now just the basics
+'''
+
+import os, sys
+import toml
+from pathlib import Path
+
+def writerpcvalues(configfile='', bitcoindusername='', bitcoindpassword=''):
```

Comment:
Not sure it's safe to set empty defaults for user/pass here.  Probably better to have an error when that happens instead?

Reply:
Its fine, if the file doesnt exist then it will bomb out. Its safe to write empty stuff.

Not to mention the entry point sets real values also.